itou/companies/models.py

```diff
@@ -363,6 +363,14 @@ def is_active(self):
     def is_opcs(self):
         return self.kind == CompanyKind.OPCS
 
+    @property
+    def is_open_to_spontaneous_applications(self):
+        return (
+            not self.block_job_applications
+            and self.spontaneous_applications_open_since is not None
+            and timezone.now() > self.spontaneous_applications_open_since
```

Comment:
Cette vérification me semble inutile. Si les `spontaneous_applications_open_since is not None`, les candidatures spontanées sont ouvertes.

Reply:
Il nous permet de ouvrir les candidatures spontanées avec une date à venir. En pratique seulement avec l'aide du site admin. C'était sympa que notre design nous permet être aussi flexible, mais ce n'est pas demandé dans la carte. Je l'enlève du coup ?

Reply:
Je l’aurai enlevé car c’est une fonctionnalité non nécessaire.